templates/applicantReports.phtml

```diff
@@ -78,6 +78,15 @@
         </table>
     </div>
 </div>
+<?php
+//echo '<pre>';
+//print_r($reports);
+//echo '</pre>';
+foreach($reports as $report) {
+    echo '<p>' . $report['report'] . ' => ' . $report['total'] . '</p>';
```

Comment:
Can you add comment that you are echoing the assoc array for testing purposes. I assume in the actual code it will need to be replaced with:
<tr>
    <td>$report['report']<td>
    <td>$report['total']<td>
</tr>

Reply:
Removed the foreach loop!